app/components/Utility/CryptoLinkFormatter.jsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import PropTypes from "prop-types";
+import QRCode from "qrcode.react";
+
+/**
+ *  Given an asset name, amount and label generates uri for its refilling
+ *
+ */
+
+class CryptoLinkFormatter extends React.Component {
+    static assetTemplates = {};
+
+    static propTypes = {
+        asset: PropTypes.string,
+        address: PropTypes.string,
```

Comment:
Asset and address are mandatory parameters

Reply:
yes, this is handled in render() method.